### ISSUE TITLE
ensemble calling feature request

### ISSUE THREAD

**Issue by parlar** [chapmanb/bcbio-nextgen#1329]
Hi!

Thank you for adding (previously) to the output which callers that actually called a variant when ensemble calling is used. I wonder if this functionality could be expanded further? Two things would be handy:
 
1. An option to include/exclude for ensemble calls variants that fail filters 
2. Report in the INFO field which filters actually failed or passed for the different callers 

cheers
Pär

**Comment by chapmanb** [comment]
Pär;
Thanks for the suggestions. I added a `use_filtered` option to the ensemble definition that allows you to include filtered calls if needed:

http://bcbio-nextgen.readthedocs.org/en/latest/contents/configuration.html#ensemble-variant-calling

Your second suggestion isn't very easy to do with the current implementation, as we tried to keep it lightweight and simple so don't have access to this when merging calls since we either remove filtered or consider them pass (depending on the inclusion option) as a preliminary step. The best bet to use this would be to use bcftools isec on the output to examine any callers where you have interest in the pass/filter status versus ensemble.

Hope this helps.
